.github/workflows/csound_builds.yml

```diff
@@ -405,6 +405,44 @@ jobs:
               cmake --build build/$ARCH --config Release
             done
 
+  wasm_build:
+    name: Wasm build (emscripten/vcpkg)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+          submodules: true
+
+      - name: Export GitHub Actions cache environment variables
+        uses: actions/github-script@v6
+        with:
+          script: |
+            core.exportVariable('ACTIONS_CACHE_URL', process.env.ACTIONS_CACHE_URL || '');
+            core.exportVariable('ACTIONS_RUNTIME_TOKEN', process.env.ACTIONS_RUNTIME_TOKEN || '');
+
+      - name: Bootstrap VCPKG
+        run: ./vcpkg/bootstrap-vcpkg.sh
+
+      - name: Install dependencies
+        run: |
+          git clone https://github.com/emscripten-core/emsdk.git
```

Comment:
https://github.com/mymindstorm/setup-emsdk Maybe try using this?

Reply:
I updated pr to use the setup-emsdk project.